modelator/src/event.rs

```diff
@@ -226,28 +226,28 @@ impl From<JsonTrace> for EventStream {
 /// You can implement several instances of [StateHandler]s
 /// and [ActionHandler]s for the `System`, thus allowing your system
 /// to handle several kinds of abstract states or actions.
-pub struct Runner<'a, System> {
-    inits: SystemTester<'a, System>,
-    actions: SystemTester<'a, System>,
-    checks: SystemTester<'a, System>,
-    expects: SystemTester<'a, System>,
+pub struct EventRunner<System: Debug> {
+    inits: SystemTester<System>,
+    actions: SystemTester<System>,
+    checks: SystemTester<System>,
+    equals: SystemTester<System>,
     outcome: String,
 }
 
-impl<'a, System> Default for Runner<'a, System> {
+impl<System: Debug> Default for EventRunner<System> {
```

Comment:
Can we remove these redundant `Default` impls and `::new()` fns. Instead derive `Default` maybe?